src/schema/associated_data.yml

```diff
@@ -0,0 +1,4 @@
+- code/
```

Comment:
YAML group recommends to use `.yaml` in favor of `.yml` so let's use `.yaml` extension